src/node_i18n.cc

```diff
@@ -204,7 +204,6 @@ class ConverterObject : public BaseObject, Converter {
 
     CHECK_GE(args.Length(), 3);  // Converter, Buffer, Flags
 
-    Converter utf8("utf8");
```

Comment:
I guess this has the side effect of verifying that ICU is in a sane state? It'll CHECK when ucnv_open() fails.

But if that's the reason why it's there, it really could use an explaining comment. cc @jasnell.

Reply:
I assumed (and still do) that this is a leftover from when this code was written, tbh.

Reply:
I think @addaleax is right